keyboards/system76/launch_1/keymaps/default/keymap.c

```diff
@@ -87,5 +90,22 @@ ________________________________________________________________________________
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case KC_LCTL:
+        lctl_pressed = record->event.pressed;
+        break;
+    case KC_RCTL:
+        rctl_pressed = record->event.pressed;
+        break;
+    case KC_ESC:
```

Comment:
This could do `if (record->event.pressed && lctrl_pressed && rctrl_pressed) system76_ec_unlock()` which should prevent the final ESC keypress from going to the OS (which I am guessing it is)

Reply:
It looks like Esc is still seen while `system76_ec_unlock()` is active (as is the rest of the keyboard), but I've added this in https://github.com/system76/qmk_firmware/pull/24/commits/a779f0c93cc738f25a5e16b87b1fc64e796616a7

Reply:
If you return false after it should not be seen.

Reply:
Aha, that did it! Amended my last commit with that change.

Side note: testing this with the Configurator doesn't work, because it continues registering the Esc key. 
![Peek 2022-03-23 15-57](https://user-images.githubusercontent.com/13512097/159802821-a316d5b0-fd84-4c8f-9798-e6bed3c2ffae.gif)

However I've confirmed in Keyboard settings that Esc isn't picked up by the OS when this hotkey is triggered.

Reply:
Correction: returning false makes it so Esc is _never_ registered by the OS. I'm going to undo that change real quick :sweat_smile: